components/trello/new-notifications.js

```diff
@@ -0,0 +1,46 @@
+const trello = require("https://github.com/PipedreamHQ/pipedream/components/trello/trello.app.js");
+
+module.exports = {
+  name: "New Notifications",
+  description:
+    "Emits an event for each new Trello notification for the authenticated user.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    trello,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
+    let notifications = [];
+    let results = [];
+
+    let since = this.db.get("lastNotificationId") || null;
+
+    let params = {
+      since,
+    };
+
+    results = await this.trello.getNotifications("me", params);
+    results.forEach(function (notification) {
```

Comment:
Wouldn't results = notifications in this case since you're just adding the same elements to a new array?